backend/template.yaml

```
@@ -1033,7 +1366,142 @@ Resources:
       TreatMissingData: notBreaching
     Condition: DeployAlarms
 
-    # SNS Topics
+  RevokeErrorsWarningAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:platform-alarms-sns-warning
+      OKActions:
+        - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:platform-alarms-sns-warning
+      AlarmDescription: !Sub
+        - 'The number of Revoke Lambda errors is greater than or equal to 10% for the latest function version. See runbook: ${WarningAlarmsRunbookUrl}'
+        - WarningAlarmsRunbookUrl: !FindInMap
+            - StaticVariables
+            - urls
+            - WarningAlarmsRunbook
+      AlarmName: !Sub ${AWS::StackName}-revoke-error-rate
+      DatapointsToAlarm: 1
+      EvaluationPeriods: 1
+      Threshold: 10
+      ComparisonOperator: GreaterThanOrEqualToThreshold
+      TreatMissingData: notBreaching
+      Metrics:
+        - Id: lambdaInvocations
+          Label: Sum of invocations for latest Lambda version
+          ReturnData: false
+          MetricStat:
+            Metric:
+              Namespace: AWS/Lambda
+              MetricName: Invocations
+              Dimensions:
+                - Name: Resource
+                  Value: !Sub ${RevokeFunction}:live
+                - Name: FunctionName
+                  Value: !Ref RevokeFunction
+                - Name: ExecutedVersion
+                  Value: !GetAtt RevokeFunction.Version.Version
+            Period: 60
+            Stat: Sum
+        - Id: lambdaErrors
+          Label: Sum of function errors for latest Lambda version
+          ReturnData: false
+          MetricStat:
+            Metric:
+              Namespace: AWS/Lambda
+              MetricName: Errors
+              Dimensions:
+                - Name: Resource
+                  Value: !Sub ${RevokeFunction}:live
+                - Name: FunctionName
+                  Value: !Ref RevokeFunction
+                - Name: ExecutedVersion
+                  Value: !GetAtt RevokeFunction.Version.Version
+            Period: 60
+            Stat: Sum
+        - Id: lambdaErrorPercentage
+          Label: Percentage of invocations that result in a function error
+          ReturnData: false
+          Expression: (lambdaErrors/lambdaInvocations)*100
+        - Id: lambdaErrorRateThreshold
+          Label: Error threshold calculation
+          ReturnData: true
+          Expression: IF(lambdaErrors >= 5,lambdaErrorPercentage)
```

suggestion: as above- 
```suggestion
          Expression: IF(lambdaInvocations >= 5,lambdaErrorPercentage)
```